### ISSUE TITLE
Provide Tap to install MTA Build Tool with Homebrew

### ISSUE THREAD

**Issue by CCFenner** [SAP/cloud-mta-build-tool#660]
To provide an easier way to install the cloud MTA build tool on MacOS or Linux machines it would be nice to have an own **Homebrew Tap** containing a formula for [Homebrew](https://brew.sh) (Package Manager for MacOS & Linux). 

SAP has already provided such a [Tap for SapMachine](https://github.com/SAP/homebrew-SapMachine).

To help myself I created a Tap containing the mbt here: https://github.com/CCFenner/homebrew-cloud-mta-build-tool

Using this one could install the latest version using `brew install ccfenner/cloud-mta-build-tool/mbt` in a standard way & ready to use.

What do you think?

**Comment by ShimiT** [comment]
This one was definitely on my to-do list  (I'm using mac either), it will be great if you can take it forward and provide a general solution via upstream. If you need my help, please let me know. 
Thank you.

**Comment by CCFenner** [comment]
👍 

I'm not sure if the *formula* can reside in this repo or needs a separate one (with respect to naming convention). Would need to try that.. 
I guess for a separate repo *you* would need to order it. But feel free to copy/fork my sample.

**Comment by ShimiT** [comment]
Yep, we need to create a new repo with `homebrew` prefix, will initiate that process.

**Comment by CCFenner** [comment]
👍 

I learned just yesterday that `mbt` requires `make` to be installed but `brew` installs `make` as `gmake`. So the dependency definition would be something one would have to look into along with the correct installation of them.

**Comment by biolauri** [comment]
Hi folks and colleagues, any news on this? :)

**Comment by ryenus** [comment]
Hi, folks, just FYI, [mbt is now available as a formula in homebrew-core][1], so you can just run `brew install mbt` and you'll have it, which should work both on mac and linux. Feel free to close this issue.

[1]: https://github.com/Homebrew/homebrew-core/blob/master/Formula/mbt.rb
